### PR TITLE
Ian branch

### PR DESCRIPTION
**Pull request by invillanueva911**

### CODE REVIEW

File: src/components/IntakeFoodContainer.js [hunk 1]
```diff
@@ -1,38 +1,140 @@
...
+                                                    //console.log(mealTitle.toLowerCase());
+                                                    //saveData('total_'+mealTitle.toLowerCase(), JSON.stringify(meal));
+                                                    setFoodArray(x); //current meal we are setting breakfast to contain
+                                                    setTotalFoodArray(meal);
+                                                    //console.log(meal.length);
+                                                    
+
+                                                    setIsDeleted(Math.random());
+                                                    setIsDelete(Math.random());
+                                                }
```
Comment (justinealvaro15) [inline src/components/IntakeFoodContainer.js]: delete unnecessary comments

File: src/components/IntakeFoodContainer.js [hunk 2]
```diff
@@ -1,38 +1,140 @@
...
+
+const IntakeFoodContainer = ({ bannerUri, food, highlight, mealTitle, navigateToSearchFood, onDeletion, onDeletion2, onDeletion3, onDeletion4, onDeletion5 }) => {
+    const meal = onDeletion5;
     const foodArray = food;
-    //console.log(food);
+    let x_date = foodArray;
+    const [isDelete, setIsDelete] = useState();
+    let counter = 0;
+    const dateSelected = onDeletion4;
+    //console.log(dateSelected);
```
Comment (justinealvaro15) [inline src/components/IntakeFoodContainer.js]: delete if not needed

File: src/components/IntakeFoodContainer.js [hunk 3]
```diff
@@ -1,38 +1,140 @@
...
+        data: '3',
+        id: '3'
+    }
+];
+
+// total_breakfast
+// total_lunch
+// total_dinner
+// total_snacks
+
```
Comment (justinealvaro15) [inline src/components/IntakeFoodContainer.js]: delete if not needed

File: src/components/IntakeFoodContainer.js [hunk 4]
```diff
@@ -1,38 +1,140 @@
-import React from 'react';
-import { FlatList, Image, StyleSheet, Text, TouchableHighlight, TouchableOpacity, View } from 'react-native';
+import React, {useState, useEffect} from 'react';
```
Comment (justinealvaro15) [inline src/components/IntakeFoodContainer.js]: add white space between imports, e.g. { useState, useEffect }

File: src/components/IntakeFoodContainer.js [hunk 5]
```diff
@@ -111,4 +213,23 @@ const styles = StyleSheet.create({
...
+        right: [
+            {
+                onPress: () => {
+
+                },
+                text: 'Delete', type: 'delete'
+            }
+        ]
+
+    };*/
```
[the remark this answers is not in the document]
Comment (invillanueva911) [reply src/components/IntakeFoodContainer.js]: dyan lng muna yan baka kasi sakaling gusto ni mam swipe delete shiz kasi dati nirequest nya yan haha ayaw ata nya sa simpleng button lng tas dat ata may confirm delete pa haha

**Pull request merged**